src/renderer/transforms/forge.ts

```diff
@@ -1,8 +1,10 @@
 import { execSync } from 'child_process';
 
 import { Files, PACKAGE_NAME } from '../../interfaces';
-import { getElectronBinaryPath } from '../binary';
+import * as path from 'path';
```

Comment:
Let's keep this in the grouping at the top with `child_process`.

Reply:
Same comment as mentioned here.
Maybe its better if I put up a thread on slack regarding it :)